src/General/Alert/Alert.test.tsx

```diff
@@ -95,12 +90,6 @@ describe('<Alert/> should render correct icon color', () => {
     const icon = queryAllByTestId('icon-svg')[0];
     expect(icon).toHaveAttribute('fill', SecondaryColor.actionblue);
   });
-
-  test('type is not a predefined type', () => {
```

Comment:
Oh why deleted these two tests?

Reply:
Hi @rosslo , oh because the Alert type only contains 4 types,
`export type AlertType = 'success' | 'warning' | 'danger' | 'info';`
thus, should we still have the test? since if we still have it it will throw an error

Reply:
Hmm, I think those test cases are to test the fallback for the wrong type value in the javascript file which doesn't enable the TypeScript. Can use `//@ts-ignore` to bypass the error.

Reply:
Hi @rosslo , Okay have updated the eslintrc.json file too to allow //@ts-ignore, ready for rereview

Reply:
Oops, sorry, I just realized, mb `<AlertComponent type={'xxx' as AlertType} />` can resolve this error too.